modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/util/ScimGroupUtil.java

```diff
@@ -24,36 +25,41 @@ public class ScimGroupUtil {
 
 	public static Group toGroup(
 			UserGroup userGroup, List<ScimUser> scimUserMembers)
-		throws Exception {
+		throws CharonException {
 
-		Group group = new Group();
+		try {
+			Group group = new Group();
 
-		for (ScimUser scimUserMember : scimUserMembers) {
-			group.setMember(ScimUserUtil.toUser(scimUserMember));
-		}
+			for (ScimUser scimUserMember : scimUserMembers) {
+				group.setMember(ScimUserUtil.toUser(scimUserMember));
+			}
 
-		group.replaceDisplayName(userGroup.getName());
+			group.replaceDisplayName(userGroup.getName());
 
-		Date createDate = userGroup.getCreateDate();
+			Date createDate = userGroup.getCreateDate();
 
-		group.setCreatedInstant(createDate.toInstant());
+			group.setCreatedInstant(createDate.toInstant());
 
-		group.setExternalId(userGroup.getExternalReferenceCode());
-		group.setId(String.valueOf(userGroup.getPrimaryKey()));
+			group.setExternalId(userGroup.getExternalReferenceCode());
+			group.setId(String.valueOf(userGroup.getPrimaryKey()));
 
-		Date modifiedDate = userGroup.getModifiedDate();
+			Date modifiedDate = userGroup.getModifiedDate();
 
-		group.setLastModifiedInstant(modifiedDate.toInstant());
+			group.setLastModifiedInstant(modifiedDate.toInstant());
 
-		group.setLocation(
-			StringBundler.concat(
-				AbstractResourceManager.getResourceEndpointURL(
-					SCIMConstants.GROUP_ENDPOINT),
-				CharPool.FORWARD_SLASH, userGroup.getPrimaryKey()));
-		group.setResourceType(SCIMConstants.GROUP);
-		group.setSchemas();
+			group.setLocation(
+				StringBundler.concat(
+					AbstractResourceManager.getResourceEndpointURL(
+						SCIMConstants.GROUP_ENDPOINT),
+					CharPool.FORWARD_SLASH, userGroup.getPrimaryKey()));
+			group.setResourceType(SCIMConstants.GROUP);
+			group.setSchemas();
 
-		return group;
```

Comment:
I think the pattern changed again since you rebased. Please see 0d2581fe77a82acc39c06876fcb3b6c7b07ede74